create_character_effects.cwt

```diff
@@ -86,6 +86,10 @@ alias[effect:enum[create_character_effects]] = {
     ### The character's religion will be this.
     religion = scope[religion]
 
+    ## cardinality = 0..1
+    ### The character's religion will be this.
+    religion = scope[character]
+

```

Comment:
Is there a way to combine ``religion = scope[character]`` and ``religion = scope[religion]``?